services/eventsServices.py

```diff
@@ -0,0 +1,108 @@
+import asyncio
+import random
+
+from entities.members import MemberDto
+
+
+class EventsServices:
+
+    async def check_barbut_command(self, bot, embed, message, payload):
+        description = embed.description.split(':')
+        stake_is = int(description[1])
+        footer = embed.footer.text.split(':')
+        player_one = int(footer[2])
+        player_two = int(footer[5])
+        allready_rolled = []
+        player_name = None
+        rolled_first_value = None
+        to_roll_player_id = player_one
+
+        if payload.emoji.name == '❌' and payload.member.id in [player_one, player_two]:
+            await message.clear_reactions()
+            return
+
+        if payload.member.id == player_one:
+            to_roll_player_id = player_two
+
+        to_roll_player = bot.guild.get_member(int(to_roll_player_id))
+
+        if payload.emoji.name == '✅' and payload.member.id == player_two:
+            await message.clear_reactions()
+            reactions = ['🎲']
+            for reaction in reactions:
+                await message.add_reaction(reaction)
+
+            # timer
+            def check(reaction, user):
+                return user in [payload.member, to_roll_player] and str(reaction.emoji) == '🎲'
+
+            try:
+                await bot.wait_for('reaction_add', timeout=30.0, check=check)
+            except asyncio.TimeoutError:
+                embed.add_field(name='Jocul a expirat', value='Nimeni nu a dat cu zarul', inline=True)
+                await message.edit(embed=embed)
+                await message.clear_reactions()
+            # timer
+
+            return
+
+        if embed.fields:
+            for field in embed.fields:
+                allready_rolled.append(field.name)
+                player_name = field.name
+                rolled_first_value = int(field.value)
+
+        if payload.emoji.name == '🎲' and payload.member.id in [player_one,
+                                                                player_two] and payload.member.display_name not in allready_rolled:
+
+            discord_member = bot.guild.get_member(payload.member.id)
+
+            roll = random.randrange(1, 100)
+            embed.add_field(name=f'{discord_member.display_name}', value=roll, inline=True)
+            await message.edit(embed=embed)
+
+            if 1 <= len(allready_rolled):
```

Comment:
This variable will never be more than 1. Better check simply if lenght is more then 0